packages/@sanity/migrate/src/mutations/creators.ts

```diff
@@ -98,6 +98,6 @@ export function delete_(id: string): DeleteMutation {
 }
 
 /**
- * Alias for delete_
+ * Alias for delete
```

Comment:
```suggestion
 * Alias for deleting a document by ID
```
A bit of a stub. Not sure if the suggestion is accurate?

Reply:
`delete` is a reserved keyword in JS, so you can't import it like this:
`import {delete} from 'sanity/migrate // syntax error`

So it's exported as both `delete_` with `del` as an alias, so you can do 
`import {del} from 'sanity/migrate // ok` or `import {delete_} from 'sanity/migrate // ok`